src/constants/upload.ts

```diff
@@ -0,0 +1 @@
+export const UPLOAD_PROVIDER = process.env.UPLOAD_PROVIDER || 'S3';
```

Comment:
o termo aqui poderia ser ASSET_STORAGE_PROVIDER, porque nao eh apenas o upload ... depois vamos ter acoes para remover arquivos por exemplo (quando um usuario for apagado).
@m-paice 

seria importante jah implementar na delecao de um usuario, a remocao de todos os seus respectivos assets e a delecao deles do ASSET_STORAGE_PROVIDER.